plugins/tracking/hooks.js

```diff
@@ -59,6 +67,37 @@ module.exports = {
     return promiseArray;
   },
 
+  pluginsConfigured() {
+    // Ensure analytics is actually ready, because in #pluginsLoadedCheck we
+    // may skip the check so that we don't completely block the applicaiton
+    const metadata =
+      Util.findNestedPropertyInObject(this, "configuration.metadata") || {};
+
+    global.analytics.ready(() => {
```

Comment:
I think i've asked this before, but where does `global.analytics` come from?

Reply:
It comes from segment.io that we're using to inject all other services.